BUILD.md

```diff
@@ -59,11 +59,11 @@ list some basic and more used commands as example.
     ```sh
     ./waf configure --board bebop --static # Bebop or Bebop2
     ./waf configure --board edge           # emlid edge
-    ./waf configure --board fmuv3          # Pixhawk2/Cube using ChibiOS
+    ./waf configure --board fmuv3          # Pixhawk 1 & 3DR Pixhawk 2 boards
```

Comment:
How about:
'Pixhawk 1 & 3DR solo (3DR Pixhawk 2) boards'

Reply:
Not anymore. The solo cube uses the CubeSolo build target. So I think the way he has it worded now is actually perfect. I wouldn't muddy up what he has in here now with solo stuff too.